CLI: Panic on tool and env subcommands

Running the bare `tk env` or `tk tool` commands panics (same stack trace for both)

Tanka 0.24.0

```
> tk tool
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x0 pc=0x1009f1414]

goroutine 1 [running]:
main.main.func1.1(0x140001ada00?, {0x1010cb778, 0x0, 0x0})
	github.com/grafana/tanka/cmd/tk/main.go:37 +0xb4
github.com/go-clix/cli.(*Command).execute(0x140002d2b40, {0x1010cb778, 0x0, 0x0})
	github.com/go-clix/cli@v0.2.0/command.go:118 +0x2b4
github.com/go-clix/cli.(*Command).Execute(0x14000261360)
	github.com/go-clix/cli@v0.2.0/command.go:76 +0xcc
main.main()
	github.com/grafana/tanka/cmd/tk/main.go:73 +0x1f8
```

Would expect a usage message listing the sub-sub-commands here!

--- a/cmd/tk/env.go
+++ b/cmd/tk/env.go
@@ -25,6 +25,7 @@ func envCmd() *cli.Command {
 	cmd := &cli.Command{
 		Use:   "env [action]",
 		Short: "manipulate environments",
+		Args:  cli.ArgsMin(2),
 	}
 
 	cmd.AddCommand(

--- a/cmd/tk/tool.go
+++ b/cmd/tk/tool.go
@@ -20,7 +20,9 @@ func toolCmd() *cli.Command {
 	cmd := &cli.Command{
 		Short: "handy utilities for working with jsonnet",
 		Use:   "tool [command]",
+		Args:  cli.ArgsMin(2),
 	}
+
 	cmd.AddCommand(
 		jpathCmd(),
 		importsCmd(),

